keras_core/ops/operation_utils.py

```diff
@@ -11,7 +11,38 @@ def compute_pooling_output_shape(
     padding="valid",
     data_format="channels_last",
 ):
-    """Compute the output shape of pooling ops."""
+    """Computes the output shape of pooling operations.
+
+    Args:
+        input_shape: Input shape. Must be a tuple of integers.
+        pool_size: Size of the pooling operation. Must be a tuple of integers.
+        strides: Stride of the pooling operation. Must be a tuple of integers.
+            Defaults to `pool_size`.
+        padding: Padding method. Available methods are `"valid"` or `"same"`.
+            Defaults to `"valid"`.
+        data_format: String, either `"channels_last`" or `"channels_first`".
```

Comment:
`"channels_lats"` or `"channels_first"`

Reply:
Hi @AakashKumarNain ,

I can see that data_format has been set as ```channels_last``` and not ```channels_lats```. Can you please confirm if I have to replace ```channels_last``` with ```channels_last``` in the docstring?

Reply:
Sorry it was typo. The main thing to change is that the string should be in quotes. I edited the comment above